### PR TITLE
Serialization Docs: add info about adding a logger

### PR DESCRIPTION
**Pull request by sebastianharko**
When writing a serializer, people want to add some debug log statements to check that their serializer is working :-) So, we need to explain how to get a logger from the ActorSystem.

IMO the best way to do this is in a comment in the code for MyOwnSerializer but wondering if there's any other opinions on this.

### CODE REVIEW

File: akka-docs/rst/scala/code/docs/serialization/SerializationDocSpec.scala [hunk 1]
```diff
@@ -19,6 +19,11 @@ package docs.serialization {
 
   //#my-own-serializer
   class MyOwnSerializer extends Serializer {
+    
+    // If you need logging here, introduce a constructor that takes an ExtendedActorSystem.
+    // class MyOwnSerializer(actorSystem: ExtendedActorSystem) extends Serializer
+    // Get a logger using:
+    // private val logger = Logging.getLogger(actorSystem, this)
```
Comment (ktoso) [inline akka-docs/rst/scala/code/docs/serialization/SerializationDocSpec.scala]: `Logging(system, this)` is scala style for that

**Review by ktoso — APPROVED** [review]
LGTM, sure why not

**Comment by sebastianharko** [comment]
@ktoso made the suggested change + added the Java part to this ...does the Java part look ok to you ? Cheers!

**Review by ktoso — APPROVED** [review]
LGTM

**Comment by sebastianharko** [comment]
@ktoso thanks!

**Review by patriknw — APPROVED** [review]
LGTM, thanks

**Comment by patriknw** [comment]
OK TO TEST

**Comment by sebastianharko** [comment]
@patriknw @ktoso I'm confused , why is this failing ?

**Comment by ktoso** [comment]
Some tests are flaky

**Comment by ktoso** [comment]
PLS BUILD

**Comment by sebastianharko** [comment]
PLS BUILD

**Comment by jlprat** [comment]
@sebastianharko you should reformat the code to follow the styleguide. Just run `sbt test:compile`.

```
+ git diff --exit-code --color
diff --git a/akka-docs/rst/scala/code/docs/serialization/SerializationDocSpec.scala b/akka-docs/rst/scala/code/docs/serialization/SerializationDocSpec.scala
index 12503fd..debb929 100644
--- a/akka-docs/rst/scala/code/docs/serialization/SerializationDocSpec.scala
+++ b/akka-docs/rst/scala/code/docs/serialization/SerializationDocSpec.scala
@@ -19,7 +19,7 @@ package docs.serialization {
 
   //#my-own-serializer
   class MyOwnSerializer extends Serializer {
-    
+
     // If you need logging here, introduce a constructor that takes an ExtendedActorSystem.
     // class MyOwnSerializer(actorSystem: ExtendedActorSystem) extends Serializer
     // Get a logger using:
+ echo '[error] Unformatted code found. Please run '\''test:compile'\'' and commit the reformatted code.'
[error] Unformatted code found. Please run 'test:compile' and commit the reformatted code.
+ false
```

A new check has been introduced recently and only correctly formatted code should be merged.

**Comment by sebastianharko** [comment]
@jlprat thanks! i'll do that!

**Comment by sebastianharko** [comment]
PLS BUILD

**Comment by sebastianharko** [comment]
@ktoso yay! test passed! this is ready to merge !

**Pull request merged**